### PR TITLE
cs_border_solid: Simplify a bit more the simple border shader.

### PR DESCRIPTION
**Pull request by emilio**
We shouldn't need to care about clip modes either, since those are only for
dotted / dashed borders.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/webrender/3022)

### CODE REVIEW

**Comment by emilio** [comment]
r? @kvark or @gw3583 

cc @djg

File: webrender/res/cs_border_solid.glsl [hunk 1]
```diff
@@ -186,29 +168,6 @@ void main(void) {
     bool in_clip_region = all(lessThan(vClipCenter_Sign.zw * clip_relative_pos, vec2(0.0)));
     float d = -1.0;
```
Comment (kvark) [inline webrender/res/cs_border_solid.glsl]: nit: later on, we don't need `d = max(d, d_radii)`

Comment (emilio) [reply webrender/res/cs_border_solid.glsl]: Nice catch, thanks :)

**Pull request merged**